staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -650,21 +673,16 @@ func (r *Request) tryThrottleWithInfo(ctx context.Context, retryInfo string) err
 	}
 	latency := time.Since(now)
 
-	var message string
-	switch {
-	case len(retryInfo) > 0:
-		message = fmt.Sprintf("Waited for %v, %s - request: %s:%s", latency, retryInfo, r.verb, r.URL().String())
-	default:
-		message = fmt.Sprintf("Waited for %v due to client-side throttling, not priority and fairness, request: %s:%s", latency, r.verb, r.URL().String())
-	}
-
 	if latency > longThrottleLatency {
-		klog.V(3).Info(message)
+		if retryInfo == "" {
+			retryInfo = "client-side throttling, not priority and fairness"
+		}
```

Comment:
`longThrottleLatency` < `extraLongThrottleLatency`, therefore we are guaranteed to enter the first if in case that any logging is taking place.

Nesting the two if statements makes that clearer and also avoids one if check in the common case that the latency is smaller than `longThrottleLatency`.

I've changed that and, while at it, also changed `var` to `const` for those two constants.